packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue

```diff
@@ -43,26 +45,39 @@ export default defineComponent({
     const anchor = ref<HTMLAnchorElement>()
 
     const { isHovered } = useHover(anchor)
-    const { getColor, getHoverColor, getTextColor, getFocusColor } = useColors()
+    const { getColor, getHoverColor, getFocusColor } = useColors()
     const { hasKeyboardFocus, keyboardFocusListeners } = useKeyboardOnlyFocus()
+    const { sidebarColor } = useSidebarItem()
+
+    const backgroundColorComputed = computed(() => {
+      if (isHovered.value) {
+        return getHoverColor(getColor(props.hoverColor || props.activeColor))
+      }
+
+      if (props.active) {
+        return getColor(props.activeColor)
+      }
+
+      if (hasKeyboardFocus.value) {
+        return getFocusColor(getColor(props.hoverColor || props.activeColor))
+      }
+
+      return getColor(sidebarColor.value)
+    })
+
+    const { textColorComputed } = useTextColor(backgroundColorComputed)
 
     const computedStyle = computed(() => {
       const style: Record<string, string> = {}
 
-      style.color = getColor(props.textColor)
+      style.color = textColorComputed.value
```

Comment:
```suggestion
      style.color = `${textColorComputed.value} !important`
```

Reply:
I think @RVitaly1978 fixed sidebar, so we don't need important anymore.